### DISCUSSION TITLE
IBM z/OS OpenTelemetry Java Instrumentation

### DISCUSSION THREAD

**Discussion by swartzkevinPFG** [open-telemetry/opentelemetry-java-instrumentation#4260] [Q&A]
We have DB2 stored procedures that run on z/OS within IBM Java.  Has anyone set up any OpenTelemetry Java Instrumentation in z/OS and had success or aware of any known issues with using OpenTelemetry Java Instrumentation on z/OS?

**Comment by iNikem** [comment]
Have you tried it? Did you encountered any problems?
